README.md

```
@@ -1 +1,38 @@
-#### my personal site
+# kumneger.dev
+
+personal website and blog, built with [Astro](https://astro.build/).
+
+---
+
+### Tech Stack
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)